### PR TITLE
[Hold] Databricks Volumes source and destination connectors for Platform

### PR DESCRIPTION
**Pull request by Paul-Cornell**
Wait for public release.

See also https://github.com/Unstructured-IO/docs/pull/272

### CODE REVIEW

File: snippets/general-shared-text/databricks-volumes-platform.mdx [hunk 1]
```diff
@@ -2,35 +2,22 @@ Fill in the following fields:
 
 - **Name** (_required_): A unique name for this connector.
 - **Host** (_required_): The Databricks workspace host URL.
-- **Cluster ID** : The Databricks cluster ID.
 - **Catalog** (_required_): The name of the catalog to use.
```
Comment (hubert-rutkowski85) [inline snippets/general-shared-text/databricks-volumes-platform.mdx:5]: Why did you remove the cluster id? It's still visible in the frontend - https://github.com/Unstructured-IO/platform-client-web/blob/b8aeb49f8bab07a1881402a084181ff7562751bd/app/globals/connectionSettingConfigs/databricksOutputSettings.ts#L94  Should it be removed from frontend?

Comment (Paul-Cornell) [reply snippets/general-shared-text/databricks-volumes-platform.mdx:5]: Things seemed to work for me without needing to specify a cluster ID.

**Comment by Paul-Cornell** [comment]
Duplicate of https://github.com/Unstructured-IO/docs/pull/327.
